split.js

```diff
@@ -143,6 +156,7 @@ Split = function (ids, options) {
             pair.gutter = gutter;
         }
 
-        el.style.width = 'calc(' + options.widths[i] + '% - ' + options.gutterWidth / 2 + 'px)';
```

Comment:
Aha! Yes! The Right Way (tm) is to adjust the **right** pane width by a **full** gutter-width, and to **not** adjust the **left** pane width **at all**!